jobs/germany.covid.forecasts/crowd-data.sh

```diff
@@ -3,6 +3,9 @@
 # check repository is up to date and update dependencies
 . schedule/jobs/germany.covid.forecasts/pull.sh
 
+# update data
+docker exec -w /home/rstudio/covid.german.forecasts forecast RscriptRscript data-raw/update.R
+
 # Redeploy forecast app (to update data)
 docker exec -w /home/rstudio/covid.german.forecasts forecast Rscript crowd-forecast/redeploy.R
```

Comment:
Shouldn't redeploy call this itself?

Reply:
there is no data update on Tuesdays. But also happy to move that into redeploy.

Reply:
hmm good point. Makes sense to have here then or have a second update bash script in crowd-forecast (I am keen on having all the code in one place so it makes sense for others as much as possible).